test-suite/misc/coq_environment.sh

```diff
@@ -9,14 +9,15 @@ cd $TMP
 cat > coq_environment.txt <<EOT
 # we override COQLIB because we can
 COQLIB="$TMP/overridden" # bla bla
+COQCORELIB="$TMP/overridden" # bla bla
 OCAMLFIND="$TMP/overridden"
 FOOBAR="one more"
 EOT
 
 cp $BIN/coqc .
 cp $BIN/coq_makefile .
 mkdir -p overridden/tools/
-cp $COQLIB/tools/CoqMakefile.in overridden/tools/ || cp $COQLIB/../coq-core/tools/CoqMakefile.in overridden/tools/
+cp $COQLIB/../coq-core/tools/CoqMakefile.in overridden/tools/
```

Comment:
Why isn't this using coqcorelib?

Reply:
Because we don't set it yet in `test-suite/Makefile` ; I was planning to clean this up in #14058 but I've chosen to address all these kind of discrepancies on #14059